website/data/docs-nav-data.json

```diff
@@ -615,36 +615,53 @@
       }
     ]
   },
-  {
-    "title": "Troubleshooting",
-    "path": "troubleshooting"
-  },
-  {
-    "title": "Glossary",
-    "path": "glossary"
-  },
   {
     "divider": true
   },
   {
-    "title": "Internals",
+    "title": "Waypoint Resources",
     "routes": [
       {
-        "title": "Overview",
-        "path": "internals"
+        "title": "Internals",
+        "routes": [
+          {
+            "title": "Overview",
+            "path": "resources/internals"
+          },
+          {
+            "title": "Architecture",
+            "path": "resources/internals/architecture"
+          },
+          {
+            "title": "Operation Execution",
+            "path": "resources/internals/execution"
+          }
+        ]
+      },
+      {
+        "title": "Roadmap",
+        "path": "resources/roadmap"
+      },
+      {
+        "title": "Community forum",
+        "href": "https://discuss.hashicorp.com/c/waypoint/"
       },
       {
-        "title": "Architecture",
-        "path": "internals/architecture"
+        "title": "Glossary",
+        "path": "resources/glossary"
```

Comment:
does the path changing require any redirects or is that done somewhere else already?

Reply:
Yeah good call! Can we also pull over these redirect definitions too?

https://github.com/hashicorp/waypoint/blob/9d06d53e4b4da92fcae1994b6e623fde867095be/website/redirects.js#L7-L26

Reply:
@BRKalow should I just copy/paste the whole nav doc? There's a lot of extra stuff in there that we don't have on stable-website.

Reply:
I'll push it up so y'all can see the diff, and then we can drop the commit if it doesn't look good.

Reply:
@krantzinator sure we can do everything here! All of the k8s pages got shifted around. Let me know if I can help!